Sprint-3/1-key-implement/2-is-proper-fraction.js

```diff
@@ -27,27 +30,37 @@ function assertEquals(actualOutput, targetOutput) {
 // Explanation: The fraction 2/3 is a proper fraction, where the numerator is less than the denominator. The function should return true.
 const properFraction = isProperFraction(2, 3);
 assertEquals(properFraction, true);
+console.log(properFraction)
 
 // Improper Fraction check:
 // Input: numerator = 5, denominator = 2
 // target output: false
 // Explanation: The fraction 5/2 is an improper fraction, where the numerator is greater than or equal to the denominator. The function should return false.
 const improperFraction = isProperFraction(5, 2);
 assertEquals(improperFraction, false);
-
+console.log(improperFraction);
 // Negative Fraction check:
 // Input: numerator = -4, denominator = 7
 // target output: true
 // Explanation: The fraction -4/7 is a proper fraction because the absolute value of the numerator (4) is less than the denominator (7). The function should return true.
```

Comment:
The function isProperFraction is not currently using the absolute value. Can you review?

Reply:
@delmorallopez - can you review your function again? It should implement using the absolute value as explained in the instructions (line 46).